### PR TITLE
add rebar3 mirror documentation

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

File: lib/hex_web/web/templates/docs/mirrors.html.eex [hunk 1]
```diff
@@ -15,7 +15,13 @@
 </p>
 
 <p>
-  <strong>Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+  <strong>Mix Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+</p>
+<p>
+  <strong>Rebar3 Example</strong>: <code>{rebar_packages_cdn, "https://s3.amazonaws.com/s3.hex.pm"}.</code>
+</p>
```
Comment (ericmj) [inline lib/hex_web/web/templates/docs/mirrors.html.eex]: I don't think it is clear where this should be configured. Is there any reason to have it over ` rebar3 hex config`?

Comment (tsloughter) [inline lib/hex_web/web/templates/docs/mirrors.html.eex]: The reason is `rebar3 hex config` only works if you have the hex plugin installed and we think you should be able to change the cdn for packages without that plugin.

It goes in the `rebar.config` file, should I just add like `Rebar3 rebar.config Example` or something.. not sure how to make it clear.

Comment (ericmj) [inline lib/hex_web/web/templates/docs/mirrors.html.eex]: Maybe we should drop the second example in that case?

We should add that it goes in `rebar.config` like you suggested and possibly add a link to rebar3 docs about configuration if you have it and if it fits on the page.

Comment (tsloughter) [inline lib/hex_web/web/templates/docs/mirrors.html.eex]: Ok, yea, just realized setting the `cdn_url` through `rebar3 hex config` doesn't really matter, it isn't used by rebar3 at all.

**Comment by tsloughter** [comment]
I still need to do some updates in the rebar3 documentation but I've added the link to it.

**Comment by ericmj** [comment]
Feel free to merge when ready.

**Pull request merged**